PoC template (#855)
kyma-project/infrastructure-manager: .github/ISSUE_TEMPLATE/poc.md

**Description**
<!-- Provide a clear and concise description of the feature. -->
This is a Proof of Concept. Here's a checklist that should help you consider often-neglected aspects:
- [ ] Should we implement a feature flag from the start?
- [ ] Should the PoC include any documentation, such as ADR?
- [ ] Is it clear who will benefit from the PoC and how?
- [ ] Will the final implementation require any migration steps?
- [ ] Is there a need for alignment with KEB/SRE/other teams? 
- [ ] Usually, it's worth timeboxing such PoC tasks. What would be a good timebox in this case?
- [ ] Will the final solution need a new repository or any pipelines adjustments?

> - [ ] are we already aware of any additional challenges related to the release process? E.g. should additional threat modelling sessions be scheduled?
> - [ ] will automated tests require any unusual efforts? E.g. complex setup for e2e tests?

Reviewer: ```suggestion
- [ ] Are we already aware of any additional challenges related to the release process? For example, should additional threat modelling sessions be scheduled?
- [ ] Will automated tests require any unusual efforts? For example, a complex setup for e2e tests?
```

**Description**
<!-- Provide a clear and concise description of the feature. -->
This is a Proof of Concept. Here's a checklist that should help you consider often neglected aspects:-

> - [ ] should we implement a feature flag from the start?
> - [ ] should the PoC include any documentation such as ADR?
> - [ ] is it clear who will benefit from the PoC and how?
> - [ ] will the final implementation require any migration steps?

Reviewer: ```suggestion
- [ ] Should we implement a feature flag from the start?
- [ ] Should the PoC include any documentation, such as ADR?
- [ ] Is it clear who will benefit from the PoC and how?
- [ ] Will the final implementation require any migration steps?
```

**Description**
<!-- Provide a clear and concise description of the feature. -->

> This is a Proof of Concept. Here's a checklist that should help you consider often neglected aspects:-

Reviewer: ```suggestion
This is a Proof of Concept. Here's a checklist that should help you consider often-neglected aspects:
```

- [ ] Should we implement a feature flag from the start?
- [ ] Should the PoC include any documentation, such as ADR?
- [ ] Is it clear who will benefit from the PoC and how?
- [ ] Will the final implementation require any migration steps?

> - [ ] is there a need for alignment with KEB/SRE/other teams? 
> - [ ] usually it's worth timeboxing such PoC tasks. What would be a good timebox in this case?
> - [ ] will the final solution need a new repository or any pipelines adjustments?

Reviewer: ```suggestion
- [ ] Is there a need for alignment with KEB/SRE/other teams? 
- [ ] Usually, it's worth timeboxing such PoC tasks. What would be a good timebox in this case?
- [ ] Will the final solution need a new repository or any pipelines adjustments?
```
